### PR TITLE
{bio}[foss/2016b] ReMatCh 3.2

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+version = '1.3.1'
+
+homepage = 'http://www.htslib.org/'
+description = """BCFtools is a set of utilities that manipulate variant calls in the
+ Variant Call Format (VCF) and its binary counterpart BCF"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/samtools/bcftools/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
```
Comment (boegel) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-foss-2016b.eb]: @pescobar missing `SHA256` checksum

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

**Review by boegel — APPROVED** [review]
old toolchain, but easy merge...

lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2603.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/70a070f6488b823da82a2a9c401dffef for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Comment by pescobar** [comment]
> old toolchain, but easy merge...

you know I am an old school guy ;)

**Pull request merged**